resources/todomvc/architecture-examples/svelte/src/utils.js

```diff
@@ -0,0 +1,7 @@
+export function uuid() {
+    return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function (c) {
+        var r = (Math.random() * 16) | 0,
+            v = c === "x" ? r : (r & 0x3) | 0x8;
+        return v.toString(16);
+    });
+}
```

Comment:
have you taken this from somewhere? If yes, can you please add a comment with a pointer? Is it worth replacing with an incremented local variable instead?

Reply:
yes, from here: https://github.com/sveltejs/svelte-todomvc/blob/master/src/TodoMVC.svelte#L64

I wanted to keep this pr as close to the original version as possible, without introducing too many changes.

Reply:
yeah I don't mind to keep it. It seemed a bit overkill to me at first, but rhere's also value in having a string.replace with a function callback in the benchmark IMO.